documentation/modules/configuring/con-config-kafka-kraft.adoc

```diff
@@ -30,35 +30,46 @@ For a deeper understanding of the Kafka cluster configuration options, refer to
 .Example `Kafka` custom resource configuration
 [source,yaml,subs="+attributes"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaApiVersion}
 kind: Kafka
 metadata:
   name: my-cluster
+# Deployment specifications
 spec:
   kafka:
+    # Kafka version (essential) 
```

Comment:
It is not required and there are situations where not setting it makes sense. So I would say it should be recommended only?